src/modules/Lessons/8_LessonUserData/components/CheckBoxField.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+
+import CheckBox from '../../../common/components/CheckBox';
+
+class CheckBoxField extends Component{
+	constructor(props){
+		super(props);
+
+		this.state = {
+			label: props.label,
+			name: props.name,
+			value: props.value,			
+		};
+		this.onChangeHandler = this.onChangeHandler.bind(this);
+	}
+
+	onChangeHandler(event){
+		this.setState({
+			value: event.target.value
```

Comment:
In the case of checkbox, we need to store all values which are checked. So you need to maintain an array instead of just single value.